### PR TITLE
adds square-root practice exercise

### PR DESCRIPTION
**Pull request by Atomic-Germ**
Changes to be committed:
	modified:   ../../config.json
	new file:   square-root/.docs/instructions.md
	new file:   square-root/.meta/config.json
	new file:   square-root/.meta/proof.ci.wren
	new file:   square-root/.meta/tests.toml
	new file:   square-root/LICENSE
	new file:   square-root/package.wren
	new file:   square-root/square-root.spec.wren
	new file:   square-root/square-root.wren

### CODE REVIEW

File: exercises/practice/square-root/.meta/proof.ci.wren [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class SquareRoot {
+
+  static squareRoot(input) {
+    var candidate = 1
+
+    if (input >= 2) {
+      while ((candidate * candidate) < input) candidate = candidate + 1
```
Comment (joshgoebel) [inline exercises/practice/square-root/.meta/proof.ci.wren:7]:
```suggestion
  static squareRoot(n) {
    var candidate = 1

    if (n >= 2) {
      while ((candidate * candidate) < n) candidate = candidate + 1
```

Comment (joshgoebel) [reply exercises/practice/square-root/.meta/proof.ci.wren:7]: `input` is such a vague name... `number` or `n`( one of the few one letter variables I think is ok) is far better.

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
@Atomic-Germ Thanks so much for another great addition! Keep em coming!

Of course the proof is NOT an efficient way of solving this, but that's not a requirement here.

**Comment by Atomic-Germ** [comment]
> @Atomic-Germ Thanks so much for another great addition! Keep em coming!
> 
> Of course the proof is NOT an efficient way of solving this, but that's not a requirement here.

😂

**Pull request merged**